tests/aat/spec/dataplane_spec.py

```diff
@@ -302,8 +308,8 @@ def do_ping(api_client, ping_binary, src_id, dst_id, domain):
         self.process = service.start()
         self.api = client.api.InterfacesApi(service.client())
 
-    with description('ipv4,'):
-        with description('ping,'):
+    with description('ipv4,', 'dataplane:ipv4'):
+        with description('ping,', 'dataplane:ipv4:ping'):
```

Comment:
mamba -t <tag>,<tag> seems to be logical OR.  so using ":" to explicitly define tag hiearchy.
mamba -t dataplane  <- Runs all dataplane tests
mamba -t dataplane:ipv4 <- Runs all dataplane IPv4 tests
mamba -t dataplane:ipv4:ping  <- Runs all dataplane IPv4 ping tests

If there is a better way to do this then LMK